### DISCUSSION TITLE
初次安装或升级到2.6.2后打开白屏？

### DISCUSSION THREAD

**Discussion by xhofe** [AlistGo/alist#1357] [Announcements]
默认使用的elemecdn已经被禁用。所以请考虑在配置文件中修改assets；配置文件位置为：
- 手动安装：运行程序的同级目录下的data文件夹中
- 一键脚本：`/opt/alist/data/config.json`
- docker: 映射目录内或进入容器内部位于`/opt/alist/data/config.json`

修改：https://alist-doc.nn.ci/docs/setting/config#assets ，其中提供的大部分已经不可用，可以考虑使用本地（即修改assets为`/`）、自行托管assets或使用`https://cdn.jsdelivr.net/npm/alist-web@$version/dist/`。

修改后重启即可。

**Comment by leoleehit** [comment]
我今天用稳定版的  也是白屏

**Comment by zhendery** [comment]
完美解决

**Comment by cxw620** [comment]
npm.elemecdn.com没缓存到就返回403了, 可能是太多人白嫖准备关掉了吧. 趁早迁移吧...

**Reply by cxw620** [reply]
> npm.elemecdn.com没缓存到就返回403了, 可能是太多人白嫖准备关掉了吧. 趁早迁移吧...

而且也不知道其缓存策略是什么, 有时候多访问几次, 大概十几分钟后就有了, 但有时候又不行

**Comment by kamasylvia** [comment]
V3 版本没有 `assets` 配置了，该怎么解决白屏问题？

**Reply by anwen-anyi** [reply]
V3 还会白屏？没见到过 截图相关内容 日志

**Reply by kamasylvia** [reply]
> V3 还会白屏？没见到过 截图相关内容 日志

我解决了，是 site_url 没设置

**Reply by wgding** [reply]
> > V3 还会白屏？没见到过 截图相关内容 日志
> 
> 我解决了，是 site_url 没设置

site_url要怎么设置？

**Reply by MingTu01** [reply]
site_url要怎么设置？

**Reply by kamasylvia** [reply]
> > > V3 还会白屏？没见到过 截图相关内容 日志
> > 
> > 
> > 我解决了，是 site_url 没设置
> 
> site_url要怎么设置？

在 `config.json` 里面设置 `site_url`，就是外网能访问你的 alist 的地址，例如：
1. `https://alist.example.com`
2. `https://www.example.com/alist`

其值取决于你设置的 CDN 和反向代理
